airmozilla/main/templates/main/_side_events.html

```diff
@@ -19,6 +19,7 @@ <h4 class="entry-title">
             </span>
             {{ event.title }}
           </a>
+        <a class="star sidebar" data-id="{{ event.id }}" title="Save to watch later"></a>
```

Comment:
Let's call it "Save as a starred event" or something. "watch later" is a different terminology.